vue-frontend/src/components/portfolio/BrandingSection.vue

```diff
@@ -0,0 +1,105 @@
+    </div>
+  </section>
+  <section class="background-image tw-relative">
+    <aspect-ratio height="56.26%">
+      <img :src="response.backgroundImage" />
+    </aspect-ratio>
+  </section>
+  <section class="tw-relative tw-bg-white">
+    <div
+      class="container tw-relative tw-pt-24 md:tw-pt-28 xl:tw-pt-52 tw-flex tw-flex-col ... md:tw-flex-row ... md:tw-gap-x-16"
+    >
+      <div class="tw-basis-1/2 tw-relative">
+        <div v-for="data in flex1" :key="data">
+          <div class="tw-relative">
+            <img :src="data.image" />
+          </div>
+          <div
+            class="v2-normal-text tw-bg-white tw-font-light ... tw-relative"
+            :class="[data.title ? 'paragraph-section' : 'image-gap']"
+          >
+            {{ data.title }}
+          </div>
+        </div>
+      </div>
+      <div class="tw-basis-1/2 md:tw-mt-40 tw-relative">
+        <div v-for="data in flex2" :key="data">
+          <div
+            class="tw-relative"
+            :style="[data.background ? { background: data.background } : '']"
+          >
+            <aspect-ratio :height="data.aspectRatio">
+              <div
+                :class="[
+                  data.video
+                    ? 'tw-px-8 tw-pb-8 md:tw-px-1 lg:tw-px-8 lg:tw-pb-8'
+                    : '',
+                ]"
+              >
+                <img :src="data.image" v-if="data.image" />
+
+                <video
+                  v-else
+                  id="video-preview"
+                  controls
+                  playsinline
+                  loop
+                  muted
+                  autoplay
+                  :class="[data.video ? 'video-border' : '']"
+                >
+                  <source :src="data.video" type="video/mp4" />
+                </video>
+              </div>
+            </aspect-ratio>
+          </div>
+          <div
+            class="v2-normal-text tw-bg-white tw-font-light ... tw-relative"
+            :class="[data.title ? 'paragraph-section' : '']"
+          >
+            {{ data.title }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+<script>
+import AspectRatio from "@/components/utils/AspectRatio.vue";
+export default {
+  props: ["response"],
+  data() {
+    return {
+      flex1: this.response.details.firstDetail,
+      flex2: this.response.details.secondDetail,
+    };
+  },
+  components: {
+    AspectRatio,
+  },
+};
+</script>
+<style lang="postcss" scoped>
+section.background-image {
+  transform: translateZ(-1px) scale(1.5);
+  z-index: -1;
+}
+.paragraph-section {
+  @apply tw-py-8  sm:tw-px-32 sm:tw-py-8 md:tw-px-14 xl:tw-p-16  2xl:tw-px-32 tw-bg-white tw-text-center;
+}
+.image-gap {
+  @apply tw-mt-12;
+}
+img {
+  @apply tw-w-full tw-h-full tw-object-cover;
+}
+.video-border {
+  @apply tw-rounded-b-[90px] tw-border-b-8 tw-border-x-8 tw-border-solid tw-border-white tw-px-8 tw-pb-8;
```

Comment:
As tailwind have concept of inlin ecss, We should add all classes inline. Remove CSS classes from CSS block

Reply:
i have used this class to apply border video conditionally , that's why i find this way appropriate.